npm/npm-debug-cli/dataplaneParser/parser/parser.go

```diff
@@ -0,0 +1,338 @@
+package parser
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+
+	"github.com/Azure/azure-container-networking/npm/npm-debug-cli/dataplaneParser/iptable"
+	"github.com/Azure/azure-container-networking/npm/util"
+)
+
+type Parser struct {
+}
+
+// CreateIptableObject create a Go object from specified iptable. Optional read from iptable-save file
+func (p *Parser) ParseIptablesObject(tableName string, filenames ...string) *iptable.Iptables {
```

Comment:
since this struct has no fields, these methods could pure functions? no need to attach them to anything